src/components/form.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from "react";
+import { Link } from "react-router-dom";
+
+export const Form = () => {
+  const [firstName, setFirstName] = useState("");
+  const [secondName, setSecondName] = useState("");
+  const [email, setEmail] = useState("");
+  const [service, setService] = useState("");
+
+  const reset = () => {
+    setFirstName("");
+    setSecondName("");
+    setService("");
+  };
+
+  const handleSubmit = (evt: React.SyntheticEvent) => {
+    evt.preventDefault();
+    const clientInfo = {
+      name: firstName,
+      surname: secondName,
+      email: email,
+      order: service,
+    };
+    localStorage.setItem("clientInfo", JSON.stringify(clientInfo));
+    reset();
```

Comment:
After saving the info, you can use `useNavigate` hook to redirect user to another page.
[Learn more here](https://reactrouter.com/en/main/hooks/use-navigate)

```tsx
const navigate = useNavigate()

...
const handleSubmit = () => {
  ...
  redirect('boooking')
}
```